subscription-engine/kafka-ws/app/WSServer.js

```diff
@@ -34,7 +33,7 @@ const getKafkaTopicFromPathname = (fullUrl, cbError) => {
   let parsedRoute = null;
   // check if the url pathname has the expected pattern with the topic
   try {
-    parsedRoute = checkAndParseURLPathname(fullUrl, '/v1/websocket/:topic');
+    parsedRoute = checkAndParseURLPathname(fullUrl, '/v1/topics/:topic');
```

Comment:
Wouldn't be better to use `/kafka-ws/v1/topicis/:topic` to be in accordance with the documentation? Or we need to explain that the service has an endpoint, but when we use the API Gateway the endpoint is different.